exercise_10/index.js

```diff
@@ -0,0 +1,11 @@
+import transformCollectionToString from "./transformCollectionToString.js";
+
+let collectionNotEmpty = transformCollectionToString([
```

Comment:
This is not an empty collection, it is a string.

```suggestion
let outputFromCollection = transformCollectionToString([
```

FIX: better use `const` if this constant will not change.